client/src/reader/components/notes/partial/GroupFilter.js

```diff
@@ -0,0 +1,71 @@
+import React, { PureComponent } from "react";
```

Comment:
I noticed I was getting an empty array for `props.readingGroups` at first, but changing this from `PureComponent` to `Component` seemed to fix it. Presumably `PureComponent` was causing it not to update when `withReadingGroups` updated that empty array with group objects.